packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2045,6 +2041,28 @@ class SemanticsNode with DiagnosticableTreeMixin {
     _children?.forEach(_redepthChild);
   }
 
+  void _updateChildMergeFlag(SemanticsNode child) {
```

Comment:
Maybe add a `recursively` to this method name to indicate that it will also do so for the child's children (similarly to other methods like that that we have in the framework).